zeus/monitor/carbon.py

```diff
@@ -191,16 +207,9 @@ class CarbonEmissionMonitor:
     measurement windows are supported.
 
     !!! Note
-                    When carbon_intensity_provider must have estimate turned on because during some hours, carbon intensity values are not recorded by ElectricityMaps.
+        carbon_intensity_provider must have estimate turned on because during some hours, carbon intensity values are not recorded by ElectricityMaps.
```

Comment:
```suggestion
        `carbon_intensity_provider` must have `estimate` turned on because during some hours, carbon intensity values are not recorded by ElectricityMaps.
```